### PR TITLE
RavenDB-16920 - Argument Null Exception when storing empty string

### PR DESCRIPTION
**Pull request by grisha-kotler**
https://issues.hibernatingrhinos.com/issue/RavenDB-16920

### CODE REVIEW

File: src/Sparrow/Extensions/EncodingsExtensions.cs [hunk 1]
```diff
@@ -9,6 +9,9 @@ internal static class EncodingsExtensions
     {
         public static unsafe int GetBytes(this UTF8Encoding encoding, ReadOnlySpan<char> chars, Span<byte> bytes)
         {
+            if (chars.IsEmpty)
```
Comment (ppekrol) [inline src/Sparrow/Extensions/EncodingsExtensions.cs:12]: Do we need to do the same in `GetString` ?

**Review by ppekrol — COMMENTED** [review]
Maybe better to move this to 5.1 as well?

File: src/Sparrow/Extensions/EncodingsExtensions.cs (hunk 1 above)
Comment (grisha-kotler) [reply src/Sparrow/Extensions/EncodingsExtensions.cs:12]: Moved PR to 5.1

**Comment by grisha-kotler** [comment]
https://github.com/ravendb/ravendb/pull/12430

**Pull request closed without merging**